### PR TITLE
Fix: Arc2D.point_belongs

### PR DESCRIPTION
**Pull request by sbendjebla**
* **Please check if the PR fulfills these requirements**
- [ ] The commit messages follows our guidelines (explicit, in english)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] CHANGELOG.md has been updated

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [ ] Bug fix:
- [ ] New features:
- [ ] Documentation
- [ ] Optimization

* **Other information**:
The old version didn't work correctly. For these arc_2d  and point_2d, it returns `True`
Even if the point_2d == arc_2d.start (or arc_2d.end)
image

### CODE REVIEW

File: volmdlr/edges.py [hunk 1]
```diff
@@ -1621,26 +1624,48 @@ def point_belongs(self, point2d, abs_tol=1e-10):
...
+        #                                                      vector_point)
+        #     else:
+        #         arc_angle = volmdlr.core.clockwise_angle(vector_start,
+        #                                                  vector_end)
+        #         point_angle = volmdlr.core.clockwise_angle(vector_start,
+        #                                                    vector_point)
+        #     if point_angle <= arc_angle:
+        #         return True
+        # return False
+
```
Comment (WirajanDASILVA) [inline volmdlr/edges.py:1668]: I believe you can delete this commented block

File: volmdlr/edges.py [hunk 2]
```diff
@@ -1621,26 +1624,48 @@ def point_belongs(self, point2d, abs_tol=1e-10):
...
-                point_angle = - volmdlr.core.clockwise_angle(vector_start,
+
+        if (self.start.point_distance(point2d) <= abs_tol
+                or self.end.point_distance(point2d) <= abs_tol):
+            return True
+
+        if self.is_trigo:
+            arc_trigo = self.reverse()
+        else:
+            arc_trigo = self
```
Comment (WirajanDASILVA) [inline volmdlr/edges.py:1635]:
```suggestion
        arc_trigo = self
        if self.is_trigo:
            arc_trigo = self.reverse()
           
```

**Review by WirajanDASILVA — CHANGES_REQUESTED** [review]

File: volmdlr/edges.py (hunk 2 above)
Comment (AxelRinghausen) [reply volmdlr/edges.py:1635]: If you really want to make this compact:
```suggestion
        arc_trigo = self.reverse() if self.is_trigo else self
```

File: volmdlr/edges.py (hunk 1 above)
Comment (sbendjebla) [reply volmdlr/edges.py:1668]: Done !

File: volmdlr/edges.py (hunk 2 above)
Comment (sbendjebla) [reply volmdlr/edges.py:1635]: Done !

**Review by masfaraud — DISMISSED** [review]

**Comment by WirajanDASILVA** [comment]
The solution proposed in [ PR merged last week](https://github.com/Dessia-tech/volmdlr/pull/485) actually works for the use case the one tries to fix. So I believe we can close this PR.

**Pull request closed without merging**